sdlib/core/stdc/stdio.d

```diff
@@ -8,3 +13,5 @@ extern(C):
 
 int printf(const char* fmt, ...);
 int puts(const char* s);
+int snprintf(char* dest, size_t size, const char* fmt, ...);
+int write(int fd, const char* buf, int count);
```

Comment:
That is defined in unistd. Why is it defined here? Can't we do it in a more standard way, such as using `fputs`?

Reply:
`stderr` is supposed to be a `FILE*`, so I don't see why this is a problem. C headers are messed up enough so that we don't need to add our mess on top of them.